addons/io/org.openhab.io.hueemulation/README.md

```diff
@@ -24,7 +24,8 @@ org.openhab.hueemulation:discoveryIp=192.168.1.100
 ```
 
 ## Device Tagging
-To expose an item on the service, apply a supported tag (which are "Lighting", "Switchable", "TargetTemperature") to it.  The item label will be used as the Hue device name.
+To expose an item on the service, apply a supported tag (which are "Lighting", "Switchable", "TargetTemperature") to it.
```

Comment:
There should be an empty line before that paragraph.

Reply:
Agree - will fix in next commit.